### PR TITLE
Add timescoring pypi package

### PR DESCRIPTION
**Pull request by danjjl**
Checklist
- [x] Title of this PR is meaningful
- [x] License file is packaged
- [x] Source is from official source
- [x] Package does not vendor other packages.
- [x] Package does not ship static libraries.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

### CODE REVIEW

File: recipes/timescoring/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "timescoring" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/timescoring-{{ version }}.tar.gz
```
Comment (martinRenou) [inline recipes/timescoring/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Review by martinRenou — COMMENTED** [review]
Just a small comment, otherwise LGTM!

File: recipes/timescoring/meta.yaml (hunk 1 above)
Comment (danjjl) [reply recipes/timescoring/meta.yaml]: Thanks for the review. I modified the recipe as per your suggestion

**Comment by danjjl** [comment]
@conda-forge/help-python, let me know if any other changes are required

**Pull request merged**